docs/platform/redirects/overview.md

```diff
@@ -0,0 +1,88 @@
+---
+id: overview
+title: Redirects
+sidebar_label: Overview
+---
+
+## Overview
+
+Redirects can be added in 2 ways. Either in a flat file (you'll need Falcon Server access for this), or, they can come from your shops API package.
+
+
+## Flat redirects (Enterprise Only)
+
+To add flat redirects you must first create a `server/config/redirects.txt` file. You can then add your redirects there. All redirects added to this file are treated as `301`.
+
+This file path can be configured in your `server/config` files. The config variable is `redirectFile`
+
+```json
+{
+  ...
+  "redirectFile": "config/redirects.txt" // default file path
+}
+```
+
+### 1 to 1 redirect
+
+The format of this file should have each redirect on a new line with the orgin and destination seperated by a space.
+
+```
+origin destination
+```
+
+**example**
+
+```
+/product-1 /product-2
+```
+
+In this example the url `/product-1` will 301 redirect to `/product-2`
+
+### Regex support
+
+We also support regex. Any entry prefixed with `@` is considered regex.
+
+**example**
+
+```
+@^\/product-1  /product-2
```

Comment:
```suggestion
@\/product-1$  /product-2
```

The current version says it should start with /product1, rather than end with it.